### ISSUE TITLE
Reply GGGGGGGGGGGGGG running nemotron:latest

### ISSUE THREAD

**Issue by 3keyallen3** [ollama/ollama#7443]
### What is the issue?

Ollama using Docker mode.
When execute 'sudo docker exec -it ollama ollama run nemotron:latest',
 or "sudo docker exec -it ollama ollama run qwen2.5:72b"
it replied "GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG" all the time.

NVIDIA-SMI 560.35.03              Driver Version: 560.35.03      CUDA Version: 12.6     |
|-----------------------------------------+------------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id          Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
|                                         |                        |               MIG M. |
|=========================================+========================+======================|
|   0  NVIDIA A100-SXM4-40GB          Off |   00000000:02:00.0 Off |                  Off |
| N/A   25C    P0             35W /  400W |   35011MiB /  40960MiB |      0%      Default |
|                                         |                        |             Disabled |
+-----------------------------------------+------------------------+----------------------+
|   1  NVIDIA A100-SXM4-40GB          Off |   00000000:61:00.0 Off |                  Off |
| N/A   26C    P0             36W /  400W |   39833MiB /  40960MiB |      0%      Default |
|                                         |                        |             Disabled |
+-----------------------------------------+------------------------+----------------------+
|   2  NVIDIA A100-SXM4-40GB          Off |   00000000:E1:00.0 Off |                  Off |
| N/A   27C    P0             37W /  400W |   37159MiB /  40960MiB |      0%      Default |
|                                         |                        |             Disabled |
+-----------------------------------------+------------------------+----------------------

### OS

Docker

### GPU

Nvidia

### CPU

Intel

### Ollama version

0.3.14

**Comment by 3keyallen3** [comment]
![图片](https://github.com/user-attachments/assets/caab501a-754e-417e-abc2-d25b1b088520)
Works well for qwen2.5:32b but qwen2.5:72b

**Comment by rick-github** [comment]
[Server logs](https://github.com/ollama/ollama/blob/main/docs/troubleshooting.md#how-to-troubleshoot-issues) might help with debugging.

```console
$ docker exec -it ollama ollama -v
ollama version is 0.3.14
$ docker exec -it ollama ollama run qwen2.5:72b
>>> hello
Hello! How can I assist you today? Feel free to ask any questions or let me know if you need help with anything specific.

>>>
$ docker exec -it ollama ollama ps
NAME           ID              SIZE     PROCESSOR    UNTIL            
qwen2.5:72b    424bad2cc13f    57 GB    100% GPU     2 hours from now    
```

**Comment by 3keyallen3** [comment]
@rick-github THX for reply.
Here is the log.
[GIN] 2024/11/01 - 00:40:13 | 200 |      32.178µs |       127.0.0.1 | HEAD     "/"
[GIN] 2024/11/01 - 00:40:13 | 200 |   15.146772ms |       127.0.0.1 | POST     "/api/show"
time=2024-11-01T00:40:13.955Z level=INFO source=sched.go:730 msg="new model will fit in available VRAM, loading" model=/root/.ollama/models/blobs/sha256-6e7fdda508e91cb0f63de5c15ff79ac63a1584ccafd751c07ca12b7f442101b8 library=cuda parallel=4 required="53.0 GiB"
time=2024-11-01T00:40:14.555Z level=INFO source=server.go:105 msg="system memory" total="2267.4 GiB" free="2236.9 GiB" free_swap="8.0 GiB"
time=2024-11-01T00:40:14.556Z level=INFO source=memory.go:326 msg="offload to cuda" layers.requested=-1 layers.model=81 layers.offload=81 layers.split=27,27,27 memory.available="[39.1 GiB 39.1 GiB 39.1 GiB]" memory.gpu_overhead="0 B" memory.required.full="53.0 GiB" memory.required.partial="53.0 GiB" memory.required.kv="2.5 GiB" memory.required.allocations="[18.6 GiB 17.1 GiB 17.3 GiB]" memory.weights.total="45.0 GiB" memory.weights.repeating="44.1 GiB" memory.weights.nonrepeating="974.6 MiB" memory.graph.full="1.3 GiB" memory.graph.partial="1.3 GiB"
time=2024-11-01T00:40:14.557Z level=INFO source=server.go:388 msg="starting llama server" cmd="/usr/lib/ollama/runners/cuda_v12/ollama_llama_server --model /root/.ollama/models/blobs/sha256-6e7fdda508e91cb0f63de5c15ff79ac63a1584ccafd751c07ca12b7f442101b8 --ctx-size 8192 --batch-size 512 --embedding --n-gpu-layers 81 --threads 96 --flash-attn --parallel 4 --tensor-split 27,27,27 --port 37747"
time=2024-11-01T00:40:14.557Z level=INFO source=sched.go:449 msg="loaded runners" count=1
time=2024-11-01T00:40:14.557Z level=INFO source=server.go:587 msg="waiting for llama runner to start responding"
time=2024-11-01T00:40:14.558Z level=INFO source=server.go:621 msg="waiting for server to become available" status="llm server error"
INFO [main] starting c++ runner | tid="140076011839488" timestamp=1730421614
INFO [main] build info | build=10 commit="b45ed63" tid="140076011839488" timestamp=1730421614
INFO [main] system info | n_threads=96 n_threads_batch=96 system_info="AVX = 1 | AVX_VNNI = 0 | AVX2 = 0 | AVX512 = 0 | AVX512_VBMI = 0 | AVX512_VNNI = 0 | AVX512_BF16 = 0 | FMA = 0 | NEON = 0 | SVE = 0 | ARM_FMA = 0 | F16C = 0 | FP16_VA = 0 | RISCV_VECT = 0 | WASM_SIMD = 0 | BLAS = 1 | SSE3 = 1 | SSSE3 = 1 | VSX = 0 | MATMUL_INT8 = 0 | LLAMAFILE = 1 | " tid="140076011839488" timestamp=1730421614 total_threads=384
INFO [main] HTTP server listening | hostname="127.0.0.1" n_threads_http="383" port="37747" tid="140076011839488" timestamp=1730421614
llama_model_loader: loaded meta data with 35 key-value pairs and 963 tensors from /root/.ollama/models/blobs/sha256-6e7fdda508e91cb0f63de5c15ff79ac63a1584ccafd751c07ca12b7f442101b8 (version GGUF V3 (latest))
llama_model_loader: Dumping metadata keys/values. Note: KV overrides do not apply in this output.
llama_model_loader: - kv   0:                       general.architecture str              = qwen2
llama_model_loader: - kv   1:                               general.type str              = model
llama_model_loader: - kv   2:                               general.name str              = Qwen2.5 72B Instruct
llama_model_loader: - kv   3:                           general.finetune str              = Instruct
llama_model_loader: - kv   4:                           general.basename str              = Qwen2.5
llama_model_loader: - kv   5:                         general.size_label str              = 72B
llama_model_loader: - kv   6:                            general.license str              = other
llama_model_loader: - kv   7:                       general.license.name str              = qwen
llama_model_loader: - kv   8:                       general.license.link str              = https://huggingface.co/Qwen/Qwen2.5-7...
llama_model_loader: - kv   9:                   general.base_model.count u32              = 1
llama_model_loader: - kv  10:                  general.base_model.0.name str              = Qwen2.5 72B
llama_model_loader: - kv  11:          general.base_model.0.organization str              = Qwen
llama_model_loader: - kv  12:              general.base_model.0.repo_url str              = https://huggingface.co/Qwen/Qwen2.5-72B
llama_model_loader: - kv  13:                               general.tags arr[str,2]       = ["chat", "text-generation"]
llama_model_loader: - kv  14:                          general.languages arr[str,1]       = ["en"]
llama_model_loader: - kv  15:                          qwen2.block_count u32              = 80
llama_model_loader: - kv  16:                       qwen2.context_length u32              = 32768
llama_model_loader: - kv  17:                     qwen2.embedding_length u32              = 8192
llama_model_loader: - kv  18:                  qwen2.feed_forward_length u32              = 29568
llama_model_loader: - kv  19:                 qwen2.attention.head_count u32              = 64
llama_model_loader: - kv  20:              qwen2.attention.head_count_kv u32              = 8
llama_model_loader: - kv  21:                       qwen2.rope.freq_base f32              = 1000000.000000
llama_model_loader: - kv  22:     qwen2.attention.layer_norm_rms_epsilon f32              = 0.000001
llama_model_loader: - kv  23:                          general.file_type u32              = 15
llama_model_loader: - kv  24:                       tokenizer.ggml.model str              = gpt2
llama_model_loader: - kv  25:                         tokenizer.ggml.pre str              = qwen2
llama_model_loader: - kv  26:                      tokenizer.ggml.tokens arr[str,152064]  = ["!", "\"", "#", "$", "%", "&", "'", ...
llama_model_loader: - kv  27:                  tokenizer.ggml.token_type arr[i32,152064]  = [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, ...
llama_model_loader: - kv  28:                      tokenizer.ggml.merges arr[str,151387]  = ["Ġ Ġ", "ĠĠ ĠĠ", "i n", "Ġ t",...
llama_model_loader: - kv  29:                tokenizer.ggml.eos_token_id u32              = 151645
llama_model_loader: - kv  30:            tokenizer.ggml.padding_token_id u32              = 151643
llama_model_loader: - kv  31:                tokenizer.ggml.bos_token_id u32              = 151643
llama_model_loader: - kv  32:               tokenizer.ggml.add_bos_token bool             = false
llama_model_loader: - kv  33:                    tokenizer.chat_template str              = {%- if tools %}\n    {{- '<|im_start|>...
llama_model_loader: - kv  34:               general.quantization_version u32              = 2
llama_model_loader: - type  f32:  401 tensors
llama_model_loader: - type q5_0:   40 tensors
llama_model_loader: - type q8_0:   40 tensors
llama_model_loader: - type q4_K:  401 tensors
llama_model_loader: - type q5_K:   40 tensors
llama_model_loader: - type q6_K:   41 tensors
llm_load_vocab: special tokens cache size = 22
time=2024-11-01T00:40:14.809Z level=INFO source=server.go:621 msg="waiting for server to become available" status="llm server loading model"
llm_load_vocab: token to piece cache size = 0.9310 MB
llm_load_print_meta: format           = GGUF V3 (latest)
llm_load_print_meta: arch             = qwen2
llm_load_print_meta: vocab type       = BPE
llm_load_print_meta: n_vocab          = 152064
llm_load_print_meta: n_merges         = 151387
llm_load_print_meta: vocab_only       = 0
llm_load_print_meta: n_ctx_train      = 32768
llm_load_print_meta: n_embd           = 8192
llm_load_print_meta: n_layer          = 80
llm_load_print_meta: n_head           = 64
llm_load_print_meta: n_head_kv        = 8
llm_load_print_meta: n_rot            = 128
llm_load_print_meta: n_swa            = 0
llm_load_print_meta: n_embd_head_k    = 128
llm_load_print_meta: n_embd_head_v    = 128
llm_load_print_meta: n_gqa            = 8
llm_load_print_meta: n_embd_k_gqa     = 1024
llm_load_print_meta: n_embd_v_gqa     = 1024
llm_load_print_meta: f_norm_eps       = 0.0e+00
llm_load_print_meta: f_norm_rms_eps   = 1.0e-06
llm_load_print_meta: f_clamp_kqv      = 0.0e+00
llm_load_print_meta: f_max_alibi_bias = 0.0e+00
llm_load_print_meta: f_logit_scale    = 0.0e+00
llm_load_print_meta: n_ff             = 29568
llm_load_print_meta: n_expert         = 0
llm_load_print_meta: n_expert_used    = 0
llm_load_print_meta: causal attn      = 1
llm_load_print_meta: pooling type     = 0
llm_load_print_meta: rope type        = 2
llm_load_print_meta: rope scaling     = linear
llm_load_print_meta: freq_base_train  = 1000000.0
llm_load_print_meta: freq_scale_train = 1
llm_load_print_meta: n_ctx_orig_yarn  = 32768
llm_load_print_meta: rope_finetuned   = unknown
llm_load_print_meta: ssm_d_conv       = 0
llm_load_print_meta: ssm_d_inner      = 0
llm_load_print_meta: ssm_d_state      = 0
llm_load_print_meta: ssm_dt_rank      = 0
llm_load_print_meta: ssm_dt_b_c_rms   = 0
llm_load_print_meta: model type       = 70B
llm_load_print_meta: model ftype      = Q4_K - Medium
llm_load_print_meta: model params     = 72.71 B
llm_load_print_meta: model size       = 44.15 GiB (5.22 BPW)
llm_load_print_meta: general.name     = Qwen2.5 72B Instruct
llm_load_print_meta: BOS token        = 151643 '<|endoftext|>'
llm_load_print_meta: EOS token        = 151645 '<|im_end|>'
llm_load_print_meta: PAD token        = 151643 '<|endoftext|>'
llm_load_print_meta: LF token         = 148848 'ÄĬ'
llm_load_print_meta: EOT token        = 151645 '<|im_end|>'
llm_load_print_meta: EOG token        = 151643 '<|endoftext|>'
llm_load_print_meta: EOG token        = 151645 '<|im_end|>'
llm_load_print_meta: max token length = 256
ggml_cuda_init: GGML_CUDA_FORCE_MMQ:    no
ggml_cuda_init: GGML_CUDA_FORCE_CUBLAS: no
ggml_cuda_init: found 3 CUDA devices:
  Device 0: NVIDIA A100-SXM4-40GB, compute capability 8.0, VMM: yes
  Device 1: NVIDIA A100-SXM4-40GB, compute capability 8.0, VMM: yes
  Device 2: NVIDIA A100-SXM4-40GB, compute capability 8.0, VMM: yes
llm_load_tensors: ggml ctx size =    1.69 MiB
time=2024-11-01T00:40:16.264Z level=INFO source=server.go:621 msg="waiting for server to become available" status="llm server not responding"
time=2024-11-01T00:40:17.417Z level=INFO source=server.go:621 msg="waiting for server to become available" status="llm server loading model"
llm_load_tensors: offloading 80 repeating layers to GPU
llm_load_tensors: offloading non-repeating layers to GPU
llm_load_tensors: offloaded 81/81 layers to GPU
llm_load_tensors:        CPU buffer size =   668.25 MiB
llm_load_tensors:      CUDA0 buffer size = 14836.62 MiB
llm_load_tensors:      CUDA1 buffer size = 14310.50 MiB
llm_load_tensors:      CUDA2 buffer size = 15398.08 MiB
llama_new_context_with_model: n_ctx      = 8192
llama_new_context_with_model: n_batch    = 512
llama_new_context_with_model: n_ubatch   = 512
llama_new_context_with_model: flash_attn = 1
llama_new_context_with_model: freq_base  = 1000000.0
llama_new_context_with_model: freq_scale = 1
llama_kv_cache_init:      CUDA0 KV buffer size =   864.00 MiB
llama_kv_cache_init:      CUDA1 KV buffer size =   864.00 MiB
llama_kv_cache_init:      CUDA2 KV buffer size =   832.00 MiB
llama_new_context_with_model: KV self size  = 2560.00 MiB, K (f16): 1280.00 MiB, V (f16): 1280.00 MiB
llama_new_context_with_model:  CUDA_Host  output buffer size =     2.45 MiB
llama_new_context_with_model: pipeline parallelism enabled (n_copies=4)
llama_new_context_with_model:      CUDA0 compute buffer size =   299.51 MiB
llama_new_context_with_model:      CUDA1 compute buffer size =   259.51 MiB
llama_new_context_with_model:      CUDA2 compute buffer size =   409.02 MiB
llama_new_context_with_model:  CUDA_Host compute buffer size =    80.02 MiB
llama_new_context_with_model: graph nodes  = 2487
llama_new_context_with_model: graph splits = 4
INFO [main] model loaded | tid="140076011839488" timestamp=1730421622
time=2024-11-01T00:40:22.841Z level=INFO source=server.go:626 msg="llama runner started in 8.28 seconds"
[GIN] 2024/11/01 - 00:40:22 | 200 |  9.618436111s |       127.0.0.1 | POST     "/api/generate"
[GIN] 2024/11/01 - 00:40:28 | 200 |  1.891222246s |       127.0.0.1 | POST     "/api/chat"
![图片](https://github.com/user-attachments/assets/fbe2e829-f89b-49a0-8ab9-6c6b1dfe5082)

**Comment by jmorganca** [comment]
This should be fixed now, but let me know if that isn't the case and we can re-open.
